### PR TITLE
feat: creates for configured regions delta changes

### PR DESCRIPTION
**Pull request by melancholiai**

### CODE REVIEW

File: src/config.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import pytz
+
+from enum import Enum
+from MapColoniesJSONLogger.logger import generate_logger
+
+app_name = 'cosmocrat'
+
+OSMOSIS_PATH='/home/amito/src/osmosis/bin/osmosis'
+OSMUPDATE_PATH='/home/amito/src/osmupdate/osmctools/src/a.out'
+OSMCONVERT_PATH='/home/amito/src/osmconvert/osmconvert'
```
Comment (syncush) [inline src/config.py:11]: there are values specified to your pc, should from env

File: src/helper_functions.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+                process_log.info,
+                handle_subprocess_exit_code,
+                (lambda: log.info('subprocess finished successfully.')))
+
+# def get_file_name_from_path(input, suffix, remove_datetime = False):
+#     format_removed = input[:input.rfind(suffix)]
+#     result = format_removed[format_removed.rfind('/') + 1:]
+#     if remove_datetime:
+#         result = remove_datetime_from_string(result)
+#     return remove_dots_from_edges_of_string(result)
```
Comment (syncush) [inline src/helper_functions.py:53]: remove if unneccesary

File: src/start.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+def initialize_environment():
+    # polygons
+    tel_aviv_polygon = Polygon('Tel Aviv', os.path.join(config.POLYGONS_PATH, 'telaviv.poly'))
+    jerusalem_polygon = Polygon('Jerusalem', os.path.join(config.POLYGONS_PATH, 'jerusalem.poly'))
+    israel_polygon = Polygon('Israel', os.path.join(config.POLYGONS_PATH, 'israel-and-palestine.poly'))
+
+    # regions
+    tel_aviv = Region(name='Tel Aviv', polygon=tel_aviv_polygon, interval=3600)
+    jerusalem = Region(name='Jerusalem', polygon=jerusalem_polygon, interval=3600)
+    israel = Region(name='Israel', polygon=israel_polygon, interval=3600, sub_regions=[tel_aviv, jerusalem])
```
Comment (syncush) [inline src/start.py:29]: should read a config and allocate the variables dynamically

**Review by syncush — CHANGES_REQUESTED** [review]

**Comment by melancholiai** [comment]
definition of done has changed.
a new pull request has been opened: https://github.com/MapColonies/cosmocrat/pull/2

**Pull request closed without merging**